src/cinder/Log.cpp

```diff
@@ -318,6 +409,44 @@ void LoggerNSLog::write( const Metadata &meta, const string &text )
 	NSLog( @"%@ %@", metaDataNs, textNs );
 }
 
+// ----------------------------------------------------------------------------------------------------
+// MARK: - LoggerNSLog
+// ----------------------------------------------------------------------------------------------------
+
+LoggerSysLog::LoggerSysLog()
+{
+	// Most general approach for determining app name.  Perhaps there is something more cinder specific?
+	// https://developer.apple.com/library/mac/qa/qa1544/_index.html
+
+	NSString *bundlePath = [[NSBundle mainBundle] bundlePath];
+	NSString *appName = [[NSFileManager defaultManager] displayNameAtPath: bundlePath];
+
+	const char *cAppName = [appName UTF8String];
+	openlog( cAppName, ( LOG_CONS | LOG_PID), LOG_USER );
+}
+
+LoggerSysLog::~LoggerSysLog()
+{
+	closelog();
+}
+
+void LoggerSysLog::write( const Metadata &meta, const string &text )
+{
+	// TODO: code duplication
+	int sysLevel;
+	switch ( meta.mLevel ) {
+		case LEVEL_VERBOSE:		sysLevel = LOG_DEBUG;	break;
+		case LEVEL_INFO:		sysLevel = LOG_INFO;	break;
+		case LEVEL_WARNING:		sysLevel = LOG_WARNING;	break;
+		case LEVEL_ERROR:		sysLevel = LOG_ERR;		break;
+
+		// debatable if FATAL is a LOG_CRIT or LOG_ALERT
+		case LEVEL_FATAL:		sysLevel = LOG_CRIT;	break;
+		default: CI_ASSERT_NOT_REACHABLE();
+	}
+	syslog(sysLevel, "%s %s", meta.toString().c_str(), text.c_str());
```

Comment:
I was just testing here and it occured to me that this priority int (`sysLevel`) should probably not ever be less than LOG_NOTICE, as it won't appear in Console / system.log unless you modify some system config files (found that info [here](http://apple.stackexchange.com/questions/13484/messages-issued-by-syslog-not-showing-up-in-system-logs)).  Our usage of `ci::log::Level` is a bit different to syslog's priority anyway - for example if `CI_MAX_LOG_LEVEL` is set to 4 then `CI_LOG_V()` calls will be compiled out (as they are in release mode by default).

That said, If I were to structure this switch statement, I would still correlate the LEVE_WARNING : LOG_WARNING, and above, but everything else would default to `LOG_NOTICE`.